src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -39,7 +39,9 @@
               <span class="error-text mx-1">This step is unfinished.</span>
               <router-link
                 :to="{ path: `/${RouteNames.DISSOLUTION_DEFINE_DISSOLUTION}` }"
-              >Return to this step to finish it</router-link>
```

Comment:
VS Code complained that this should be on a separate line.

However, that means the text has a leading and trailing carriage return (which is typically ignored by the renderer/browser), shich is a pain to match in unit tests. By wrapping the text in a span, unit testing is easier.